beesdoo_shift/wizard/subscribe.py

```diff
@@ -134,3 +134,23 @@ def subscribe(self):
             # ,...
             status_id.sudo().write(data)
         return True
+
```

Comment:
Since the shift is replaced there is no sense to show this warning. A better approach would be to show the as default value the shift the worker already subscribed